### ISSUE TITLE
Installer never asks me to choose my region and language

### ISSUE THREAD

**Issue by vanvugt** [canonical/ubuntu-desktop-installer#2324]
### What happened?

Installer never asks me to choose my locale so it installs US English. This seems to be a recent regression?

### What was expected?

I get asked about my region and language so I can set Australian English.

### Steps to reproduce

1. Install Ubuntu 23.10.
2. Reboot.

### Additional context

Bug first encountered in image 2023-08-29 and confirmed again today in 2023-09-04.

**Comment by seb128** [comment]
There is something wrong with the selection of pages, here the 'installer only' mode doesn't display the keyboard selection step but the page is there when starting the installer from the live session...

**Comment by seb128** [comment]
In fact trying again today it seems to work correctly. The first page displayed in the language selection, isn't that the case for you? Was the behaviour different in lunar?

**Comment by vanvugt** [comment]
The language page only contains English (which means US English). There's no page to configure Australian English like there used to be.

**Comment by gunnarhj** [comment]
I was kind of caught of guard when realizing that this aspect of the installer hasn't been properly addressed yet.

Please note that in Ubiquity the time zone location is used for "guessing" the desired locale. The tz setting is used both for picking a proper English locale, but also for setting regional formats explicitly when the user has picked a language which does not match the tz setting (typical use case: user wants English for display language but their local date etc. formats).

That behavior is sync'ed with gnome-language-selector, and — if I recall correctly — the reason why some options in gnome-initial-setup are hidden in Ubuntu.

Expanding the language list would allow the user to pick the desired English variant, but it would not deal with regional formats.

**Comment by getsnoopy** [comment]
Yes, as mentioned in #2386, I'm not sure who's making these decisions, but US English is probably the worst choice as a default. Depending on which metrics are being measured, it is an inappropriate choice for 74%, 96%, and ~65% of the world for the English variant, date format, and clock format, respectively.

It needs to be changed to British English by default.
